frontend/src/services/utils.js

```diff
@@ -0,0 +1,23 @@
+export async function fetchStudent(student_ID) {
+  try {
+    const url = "http://localhost:8000/fetch";
```

Comment:
```suggestion
    const url = `${process.env.BACKEND_URL}/fetch`;
```

Reply:
To keep testing & development easier, I suggest configuring these later when setting up for deployment